lib/broccoli-build-error.js

```diff
@@ -43,6 +43,15 @@ function BroccoliBuildError(originalError, node) {
   this.broccoliPayload.error.location.file = originalError.file;
   this.broccoliPayload.error.location.treeDir = originalError.treeDir;
 
+  var location = originalError.location || originalError.loc;
```

Comment:
could you maybe incorporate these changes into `broccoli-middleware` as well?

Reply:
I believe these changes will end up in the `broccoli-middleware`. I have tested and it now shows the line info where it was not before.

For reference: https://github.com/ember-cli/broccoli-middleware/blob/master/lib/utils/error-handler.js#L32